codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/message/MessageMarshallableGenerator.kt

```diff
@@ -70,7 +70,19 @@ class MessageMarshallableGenerator(
                                     eventPayloadBinding != null -> renderSerializeEventPayload(ctx, eventPayloadBinding, writer)
                                     unbound.isNotEmpty() -> {
                                         writer.addStringHeader(":content-type", payloadContentType)
-                                        renderPayloadSerialization(ctx, writer, variant)
```

Comment:
Added support here for unbound event members, which (according to the Smithy spec) are supposed to be serialized by
> combining all members that are not marked with the eventHeader or eventPayload trait into a protocol-specific document
(https://smithy.io/2.0/spec/streaming.html#event-message-serialization)

So, I do that here.